### DISCUSSION TITLE
SDK SES - Sending header with AwsRequestOverrideConfiguration

### DISCUSSION THREAD

**Discussion by ghost** [aws/aws-sdk-java-v2#4976] [Q&A]
I'm trying to send the List-Unsubscribe header with the SES sdk, but it is ignored when sending.
Estou usando Spring Boot 3 + Java 17

`

        AwsRequestOverrideConfiguration overrideConfig = AwsRequestOverrideConfiguration.builder()
                .putHeader("List-Unsubscribe", "<https://.../mail/unsubscribe/xxxxxxxxxxxxxxx>")
                .build();

        SendEmailRequest request = SendEmailRequest.builder()
                .destination(Destination.builder().toAddresses(sMailTo).build())
                .message(Message.builder()
                    .body(Body.builder()
                        .html(Content.builder().charset("UTF-8").data(bodyHTML).build())
                        .text(Content.builder().charset("UTF-8").data(bodyText).build())
                        .build())
                    .subject(Content.builder().charset("UTF-8").data(subject).build())
                    .build())
                .source(sMailFrom)
                .configurationSetName(null)
                .overrideConfiguration(overrideConfig)
                .build();
            
        client.sendEmail(request);
`

**Comment by visa2learn** [comment]
Did you figure this out? I am facing similar issue with headers set via overrideConfiguration being ignored
